content/en/blog/_posts/2023-01-05-retroactive-default-storage-class.md

```diff
@@ -11,7 +11,7 @@ The v1.25 release of Kubernetes introduced an alpha feature to change how a defa
 StorageClass was assigned to a PersistentVolumeClaim (PVC). With the feature enabled,
 you no longer need to create a default StorageClass first and PVC second to assign the
 class. Additionally, any PVCs without a StorageClass assigned can be updated later.
-This feature was graduated to beta in Kubernetes v1.26.
+This feature was graduated to beta in Kubernetes v1.26 and GA'ed in v1.28.
```

Comment:
```suggestion
This feature was graduated to beta in Kubernetes v1.26. _Update: this feature has graduated to GA in v1.28._
```

Reply:
@reylejano Added your suggestion and squashed, thank you!